zyte_common_items/util.py

```diff
@@ -2,7 +2,7 @@
 from typing import Any, Callable, Dict, List, Optional, Tuple, Type
 from weakref import WeakKeyDictionary
 
-import attr
+import attrs
```

Comment:
It might be good to define minimum attrs version in setup.py, see https://www.attrs.org/en/stable/names.html#tl-dr - probably it should be 21.3.0